app/src/main/java/com/amaze/filemanager/ui/notifications/FtpNotification.java

```diff
@@ -49,46 +48,23 @@ private void createNotification(Context context, boolean noStopButton) {
                 + address.getHostAddress() + ":"
                 + port + "/";
 
+        NotificationCompat.Builder builder = FtpService.getBuilder();
```

Comment:
How are you sure that `FtpService.buildNotification()` was called and builder is not null? IMO it should be a compile time fail if it was not called for some reason, not a runtime crash.

Reply:
The reason I made the NotificationCompat.Builder builder object static, the getter for the builder object static, and this method call all the way they were is that I couldn't figure out how to get the notification builder from FtpService to FtpNotification. However, after reviewing this today I now see that none of this needs to be static and I can update the notification that is always started in FtpService in FtpNotification by using the FTP_CHANNEL_ID. 

I am still testing my changes, but I know that the update to the notification in FtpNotification will always happen after FtpService notification is created because:
1. The notification is created in FtpService's onStartCommand callback
2. FtpNotification is only updated with a broadcast is sent to FtpNotification after the FTP Server is started and running. 

I should have listened to my gut because making the above static and calling it in FtpNotification just didn't feel right.